app/src/main/java/com/vci/vectorcamapp/landing/presentation/components/LandingActionTile.kt

```diff
@@ -55,6 +60,23 @@ fun LandingActionTile(
                     tint = MaterialTheme.colors.icon,
                     modifier = Modifier.size(MaterialTheme.dimensions.iconSizeLarge)
                 )
+
+                if (badgeCount > 0) {
+                    Badge(
+                        modifier = Modifier
+                            .align(Alignment.TopEnd)
+                            .offset(
+                                x = MaterialTheme.dimensions.paddingExtraSmall,
+                                y = -MaterialTheme.dimensions.paddingExtraSmall
+                            ),
+                        containerColor = MaterialTheme.colors.error
+                    ) {
+                        Text(
+                            text = badgeCount.toString(),
+                            color = MaterialTheme.colors.buttonText
+                        )
+                    }
+                }
             }
```

Comment:
Yeah badge should probably not belong here. You should have it in the landing screen. If in the future we add actions which require us to add more badges, we can consider moving it in here.